CONTRIBUTING.md

```diff
@@ -237,6 +237,12 @@ $ git commit --amend
 $ git push --force-with-lease origin my-branch
 ```
 
+When the commits in your Pull Request get landed, they will be squashed into
+one commit, with metadata added to the commit message (including links to the
+Pull Request and the names of the reviewers). This will be done by someone who
+has the commit access to the repository. The commit history of your Pull
+Request, however, will stay intact on the Pull Request page.
```

Comment:
This isn't actually true, the commit history of your PR is available until you delete your fork branch, at which point it disappears.